chkconfig.spec

```diff
@@ -99,7 +97,6 @@ mkdir -p $RPM_BUILD_ROOT/etc/chkconfig.d
 %{_sbindir}/alternatives
 %{_mandir}/*/update-alternatives*
 %{_mandir}/*/alternatives*
-%dir /var/lib/alternatives
```

Comment:
I think this may need to become `%ghost` or so? Or it could continue to always exist and we just ignore it for image based builds.